website/www/site/content/en/community/powered-by.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Powered by Apache Beam'
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Projects and Products Powered by Apache Beam
+
+To add yourself to the list, please open a [pull request](https://github.com/apache/beam/edit/master/website/www/site/content/en/community/powered_by.md) adding your organization name, URL, a list of which Beam components you are using, and a short description of your use case.
+
+* **[Cloud Dataflow](https://cloud.google.com/dataflow):** Google Cloud Dataflow is a fully managed service for executing
```

Comment:
Can we make this content come from a yaml? I'm thinking we can have the following fields:
- name
- link
- description

This will make it easier for people to contribute, and also for us to change the format in which the "powered by" data shows up in the future.

Reply:
We can. Do you have an example of it somewhere?

I copied this mostly from https://arrow.apache.org/powered_by/ - and I think the all in one single page was friendly for people to come and edit.

Reply:
You can see an example in https://github.com/apache/beam/pull/11802 -- basically, you'd need to create the yaml file under www/site/data, create a shortcode in www/layouts/shortcodes, and then render that shortcode (with something like `{{< io-matrix >}}`).

The shortcode template code would likely be much simpler in this case than in that PR, though, since we're just rendering a bulleted list.

Reply:
This actually sounds complicated then keeping a single page. Let's first see if we have enough projects/products to add this page. We can iterate if we agree to have this page in.

Reply:
It wouldn't be too bad -- I'd be glad to help do it -- it would just make sure that we maintain a consistent format and layout that's easy to change (for example, https://arrow.apache.org/powered_by/ isn't in alphabetical order, and the bolding of titles etc. isn't too consistent -- and it would take more than a simple change to change this format quickly, because it may not be data-driven like with YAML). Making new data templates in YAML will also help us be better prepared for localization down the road.

If you're okay with it, we could probably do it after this PR is merged.